portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -94,4 +102,11 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Redirect back to the HTML forum page.
     response.sendRedirect("/forum.html");
   }
+
+  public Comment createComment(Entity entity) {
```

Comment:
I think pulling this out was a readability improvement. You may want to make this private unless you intend to use it from other classes (as a general rule).